gapic-generator/templates/default/client/method/def/_normal.erb

```diff
@@ -49,15 +49,7 @@ def <%= method.name %> request = nil, options: nil, **request_fields, &block
   request ||= request_fields
   request = Google::Gax.to_proto request, <%= method.request_type %>
 
-  # Converts hash and nil to an options object
-  options = Google::Gax::ApiCall::Options.new options.to_h if options.respond_to? :to_h
-
-  # Customize the options with defaults
-  header_params = {} # { name: request.name }
-  request_params_header = header_params.map { |k, v| "#{k}=#{v}" }.join("&")
-  metadata = @metadata.merge "x-goog-request-params".freeze => request_params_header
-  retry_policy = {} # retry_codes: [GRPC::Core::StatusCodes::UNAVAILABLE] }
```

Comment:
@jbolinger I moved the line with the comment as-is from the `method/def` templates. @blowmage can we remove this comment?

Reply:
That will get removed when we implement retry codes. Let's leave it there for now.